playbooks/roles/install-docker/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# defaults file for install-docker
+
+# We don't know which default python version will be configured on the remote
+# node, therefore we install both docker packages, so we'll be able to find
+# the python libs anyway.
+docker_packages:
+  - docker
+  - python2-docker
+  - python3-docker
```

Comment:
Actually I think if you install just `python-dockerpty` it should pick the right version

Reply:
Mmm what is "the right" version, as it might totally be defined in userspace? I don't know how python-dockerpty is installed, neither if it provides the docker library needed for https://docs.ansible.com/ansible/2.7/modules/docker_container_module.html#docker-container-module . But I would be happy to do a follow up patch to clean that up. I just know this works for now :)

Reply:
The right version is either python2-docker or python3-docker, I guess you do not need both.

It might be better to just use `python-docker` actually.

Reply:
oh python-docker is pointing to the right default python version?
That would be neat.

If someone is not running dual stack python, it means we are installing a package too much here. But that's not really the problem. The problem is that we can't predict reliably what will be used for the python on that machine because:
- Some people could have made the system dual stack by installing python3 (IIRC 15 is still python2 by default) 
- The deployer can decide with python version to use in the inventory
- Ansible will fallback to local ansible's python version if nothing is defined. Which can be python3 for a modern OS, or python2 for a less modern one :p . But we can't predict that -- IIRC.

What we could do instead, is have a stance: "Please install x on this machine, and configure inventory appropriately"

Reply:
Edit: 15 has no python by default in my container, so I guess we can use whatever we want. It's probably worth testing though. Can we do that in a different PR?

Reply:
I'm fine with that

Reply:
Solved in PR97.